src/udev/udev-builtin-net_id.c

```diff
@@ -45,6 +45,7 @@
  *                                         — PCI geographical location
  *   [P<domain>]p<bus>s<slot>[f<function>][u<port>][..][c<config>][i<interface>]
  *                                         — USB port number chain
+ *   v<slot>                               - VIO slot number
```

Comment:
A good summary/overview of IBM PowerVM is the following Redbook
[http://www.redbooks.ibm.com/abstracts/sg247940.html](url)

Chapter 17 of the following LoP draft document contains deeper details on the Virtual I/O layer and handling:
[https://openpowerfoundation.org/wp-content/uploads/2016/05/LoPAPR_DRAFT_v11_24March2016_cmt1.pdf](url)